app/views/plates/_choose_workflow.html.erb

```diff
@@ -1,10 +1,17 @@
 <%= vite_javascript_tag 'entrypoints/pages/choose_workflow.js', defer: true %>
 <%= card title:'Choose workflow', id: 'choose_workflow_card', css_class: 'suggested-actions logged_in_only' do %>
   <p class="lead">
+  <%# TODO: confirm that this is a suitable check and does not disrupt workflows besides LCMT WGS branch %>
+  <% if not presenter.labware.any_complete_requests? %>
```

Comment:
I don't see any checking of requests working on a labware created via permissive presenters, because I think that labware will be in pending state and have no aliquots/samples or requests because nothing has been transferred into it yet.
(we discussed this the other day, but I forget the conclusion! am I wrong?)

In theory a plate in a pipeline could have both completed and active requests. e.g. Bespoke chromium splits into 3 library preps each with a different submission and requests. So, including the closed request for the DNA prep part of the pipeline that ends with creation of the branch point plate, the branch plate could have 3 more to make 4 in total, e.g. 2 completed sets of library prep requests per well plus a third set of library prep requests still active (3 different library types). (NB. that one does not have automated submissions)